.github/workflows/update-types.yml

```diff
@@ -12,6 +12,13 @@ on:
       single_path:
         description: 'The path to generate types for (e.g. "compute", or "keyvault"). Leave blank to generate all types.'
         required: false
+      oci_artifact_version:
+        description: 'The version to be used to publih the types provider OCI artifact.'
+        required: true
```

Comment:
Can we use nbgv instead to generate the version automatically?

Reply:
we an but since we are waiting for a resolution on what the version needs to be for az packages, for now I am using `0.0.0-alpha-${short-sha}` this way the change becomes easy once the team decides what the az version needs to be and how its calculated.